### DISCUSSION TITLE
npm run spritesheet

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#4545] [Hubs Cloud Questions]
I have an error when I run 'NPM run spritesheet'. Does anyone know about this problem  
![1629349668(1)](https://user-images.githubusercontent.com/69456381/130011324-aa03c6bb-78c7-4ae5-a62b-6b9e30946fd4.png)

**Comment by pattersonbl2** [comment]
What install type is this ?

**Reply by lihong-xmuseum** [reply]
Modify the image in Spritesheet, and then execute this command to update the JSON and PNG
